docs/spark-qualification-tool.md

```diff
@@ -493,6 +493,82 @@ details on limitations on UDFs and unsupported operators.
 11. _Exec Children Node Ids_
 12. _Exec Should Remove_: whether the Op is removed from the migrated plan.
 
+**Parsing Expressions within each Exec**
+
+The Qualification tool looks at the expressions in the _Exec_ to extract any duration information.
+Note that it is not possible to extract the expressions for each available _Execs_:
+- some Execs do not take any expressions, and 
+- some execs may not show the expressions in the eventlog. For example:
+
+  - `SortMergeJoin` shows only the columns
+    ```
+    df1.select(ceil(col("num")) as "a")
+       .join(df2.select(ceil(col("num")) as "b"), $"a" === $"b"))
```

Comment:
I want to be clear, what are we trying to get across here?  I think we can assume that its doing an "===" here against the 2 columns listed. Is there something that we check in our tagging code that we would determine this can't go on the GPU?  If not then this doesn't really matter.   For instance I know we check the type of join, inner, outer, etc.  I didn't think there was any other checks for joins other then data types.  

The other things like ceil are handled by project and filters. 

I just want to make sure we aren't confusing the user.